.npmignore

```diff
@@ -0,0 +1,5 @@
+*.png
+.github/
+docsify/
+.nyc_output/
+coverage/
```

Comment:
This is why we haven't done it before: these are already covered by `.gitignore`. With this PR merged, we'll have to maintain overlap. I wish `.npmignore` would augment instead of replace.